### DISCUSSION TITLE
Add Ability To Create Meta-Packages Through Website UI

### DISCUSSION THREAD

**Discussion by khalidabuhakmeh** [NuGet/Home#10531] [Ideas]
Meta-packages allow folks to deliver multiple transitive dependencies to users who install the _one_ package. As a user, I want to be able to define a meta-package and publish that _"recipe"_ into the NuGet feed.

I could imagine meta-packages having the Id of the individual or organization that defined it. For example, these could be my personal meta-packages.

```
khalidabuhakmeh/aspnetcore-web
khalidabuhakmeh/efcore-sql-server
khalidabuhakmeh/blazor-wasm
```

These meta-packages could have a package type of `recipe` or something along those lines.

**Comment by JonDouglas** [comment]
How I imagine this would be a `Package Bundle` or similar. The best explanation I can picture would be an [App Bundle](https://developer.apple.com/app-store/app-bundles/) similar to how you might buy a bundle & receive multiple unique applications that are installed on your phone.

This of course would rather be a `Package Bundle` that would install multiple unique packages to your project/solution.

What's your thoughts?

**Reply by JonDouglas** [reply]
I also stumbled upon a cool resource that utilizes this concept in the Rust ecosystem. See this cookbook https://rust-lang-nursery.github.io/rust-cookbook/intro.html which takes a group of crates to accomplish a recipe.

**Reply by khalidabuhakmeh** [reply]
Yes! This would be very cool. Thanks for finding the Rust `crate` example.

**Reply by JonDouglas** [reply]
This seems related to this discussion: https://github.com/NuGet/Home/issues/1089
